sources/api/migration/migrations/v1.10.0/dns-settings/src/main.rs

```diff
@@ -0,0 +1,24 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::AddPrefixesMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added new settings under `settings.dns` for configuring /etc/resolv.conf
+fn run() -> Result<()> {
+    migrate(AddPrefixesMigration(vec![
+        "settings.dns",
+        "services.dns",
+        "configuration-files.netdog",
```

Comment:
Now that we've change the name to `netdog-toml`, although this will still work, I think it'd be better to be more specific.
```suggestion
        "configuration-files.netdog-toml",
```

Reply:
:)  Thanks for pointing that out - I was going back through to make sure I hadn't missed anything and didn't catch that yet.